File: module_me/src/main/java/com/fly/tour/me/contract/NewsDetailAddContract.java
package com.fly.tour.me.contract;

import com.fly.tour.common.mvp.BaseView;

/**
 * Description: <NewsDetailAddContract><br>
 * Author:      mxdl<br>
 * Date:        2019/5/27<br>
 * Version:     V1.0.0<br>
 * Update:     <br>
 */
public interface NewsDetailAddContract {
    interface Presenter{
        void addNewsDetail(int type,String title,String content);
    }
    interface View extends BaseView {}
    interface Model{
        boolean addNewsDetail(int type,String title,String content);
    }
}

File: plugin_fly_me/src/main/java/com/fly/tour/me/contract/NewsDetailAddContract.kt
package com.fly.tour.me.contract

import com.fly.tour.common.mvp.BaseView

/**
 * Description: <NewsDetailAddContract><br>
 * Author:      mxdl<br>
 * Date:        2020/2/17<br>
 * Version:     V1.0.0<br>
 * Update:     <br>
 */
interface NewsDetailAddContract {
    interface Presenter {
        fun addNewsDetail(type: Int, title: String, content: String)
    }

    interface View : BaseView {}
    interface Model {
        fun addNewsDetail(type: Int, title: String, content: String):Boolean
    }

}